packages/app-web/src/slices/ui/index.ts

```diff
@@ -119,6 +119,10 @@ export const settingsSlice = createSlice({
     }>) => {
       pushModal(state, payload.payload)
     },
+    toggleAddModalAction: (state, { payload }: PayloadAction<{}>) => {
+      if (state.modalStack.length === 1 && state.modalStack[0].type === 'add') popModal(state)
+      else if (state.modalStack.length === 0) pushModal(state, { type: 'add' })
```

Comment:
Please always add curly braces around an `if`, `else if`, or `else` body and move embedded statements into new lines. I noticed, that this needs to be added to the JavaScript styleguide as well.

Apart from that this code is really clear and concise. Awesome!

Reply:
Huh, I thought the style guide was based on the rules set by JS-standard? Also, any clarifications on the use of arrow comments and how to structure them? Because there are multiple ways of structuring them

Reply:
Also, does the else go on a new line or same line as the closing curly brace for the first if?

Reply:
Yes, we're using the standard JS style. I learned yesterday that I need to put some custom rules on top, as I'd like to enforce curly braces in this situation. What do you mean by 'arrow comments'? The way you formatted the if structure is fine.